storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2138,7 +2138,7 @@ static my_bool
 innodb_init(void)
 {
 	int	err;
-
+  os_event_global_init();
```

Comment:
alignment here and other palces too.. I guess it uses tabs instead of spaces

Reply:
@satya-bodapati fixed